applications/nrf5340_audio/src/bluetooth/ble_hci_vsc.h

```diff
@@ -56,6 +66,13 @@ struct ble_hci_vs_cp_set_radio_fe_cfg {
 	uint8_t ant_id;
 } __packed;
 
+enum ble_hci_vs_max_tx_power {
+	BLE_HCI_VSC_MAX_TX_PWR_0dBm = 0,
+	BLE_HCI_VSC_MAX_TX_PWR_3dBm = 3,
+	BLE_HCI_VSC_MAX_TX_PWR_10dBm = 10,
+	BLE_HCI_VSC_MAX_TX_PWR_20dBm = 20
+};
+
 enum ble_hci_vs_tx_power {
```

Comment:
Maybe add a comment here that "higher values are possible, but then it gets more complicated"

Reply:
Seems the max output power for nRF21540 is 20 dBm.

Reply:
Seems the max output power for nRF21540 is 20 dBm.

Reply:
I'd rather not open that can of worms until we have discussed this further.